### PR TITLE
CI: enable stale as a reminder or housekeeping

### LINKED ISSUE #326: Add stale bot https://github.com/probot/stale
**Is your feature request related to a problem? Please describe.**
Use stale bot to do house keeping

**Describe the solution you'd like**
So far we have many issues open, and I found some of them keep open for lot of days.
Hence, by this bot, I want to have a reminder or housekeeping for issues.

**Describe alternatives you've considered**
n/A, welcome for any others.

**Additional context**
n/A, welcome for any others.

### PR DESCRIPTION
**Pull request by SamYuan1990**
resolve #326 

Signed-off-by: Sam Yuan <yy19902439@126.com>

### CODE REVIEW

**Comment by SamYuan1990** [comment]
so far use default settings of stale bot.

**Review by marceloamaral — APPROVED** [review]
/lgtm

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels: []
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
```
Comment (rootfs) [inline .github/stale.yml:14]: @SamYuan1990 can you have a followup PR to document the labels?

Comment (SamYuan1990) [reply .github/stale.yml:14]: [updated at](https://github.com/orgs/sustainable-computing-io/projects/2/views/1)

**Pull request merged**